haskell/experimental/private/module.bzl

```diff
@@ -548,6 +569,7 @@ def build_haskell_modules(
             object files and the object files of their transitive module
             dependencies. See Note [Narrowed Dependencies].
         per_module_transitive_dyn_objects: like per_module_transitive_objects but for dyn_o files
+        repl_info: struct(source_files, boot_files, import_dirs, user_compile_flags)
```

Comment:
Is this enough documentation? If not, how should I document the fields of the returned struct?

Reply:
I would add some, to match the style of the doc for the other arguments. It looks you can indent it as it is done [here](https://github.com/tweag/rules_haskell/blob/master/haskell/private/path_utils.bzl#L101-L106).

Reply:
Added some docs here.